doc/user/content/transform-data/patterns/time-travel-queries.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Time travel queries"
+description: "How to use history retention to enable time travel queries in Materialize"
+menu:
+  main:
+    parent: 'sql-patterns'
+    name: "Time travel queries"
+---
+
+{{< private-preview />}}
+
+By default, all user-defined sources, tables, materialized views, and indexes
+keep track of the most recent version of their underlying data. Some use cases
+require accessing past versions of this data at specific points in time
+(i.e. time traveling). To allow time travel queries, you can configure these
+objects to **retain history**.
+
+When set to a value different than the default (i.e. no history retention), the
+history retention period of an object determines the duration for which
+historical versions of its underlying data are available for querying.
+
+The common use cases for time traveling in Materialize are:
+
+* Lossless, **durable subscriptions** to your changing results. See the
```

Comment:
Perhaps as a followup: in the past we've talked about organizations that can't handle the continuous nature of SUBSCRIBE. We provide DECLARE + FETCH as a way around this. Using that should be a third bullet point here, with fuller descriptions maybe on the FETCH page.

Reply:
Agreed! If you don't mind, I'll mark that as a follow up since we're already covering so much in this initial pass.

Reply:
Added a todo in the docs as a placeholder!